src/phase0/configs/mainnet.rs

```diff
@@ -1,14 +1,16 @@
 use crate::primitives::{Epoch, Gwei, VersionBytes};
+use ssz_rs::Vector;
 
 pub const MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: u64 = 16384;
 pub const MIN_GENESIS_TIME: u64 = 1606824000;
 pub const GENESIS_FORK_VERSION_BYTES: VersionBytes = [0u8; 4];
```

Comment:
we can drop `GENESIS_FORK_VERSION_BYTES` if we figure out how to declare `GENESIS_FORK_VERSION: Version` as a constant

Reply:
what about just
`pub const GENESIS_FORK_VERSION: [u8; 4] = [0u8; 4];`

Reply:
yeah i think this was before i added the `[u8;N]` `SimpleSerialize` impls, agree that we can drop the `GENESIS_FORK_VERSION_BYTES` thing (that was just going to be a workaround)